lib/transformations/__snapshots__/utils.test.js.snap

```diff
@@ -1,24 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`utils createLiteral should create basic literal 1`] = `
-Object {
-  "comments": null,
-  "loc": null,
-  "regex": null,
-  "type": "Literal",
-  "value": "strintLiteral",
-}
-`;
+exports[`utils createLiteral should create basic literal 1`] = `"\\"stringLiteral\\""`;
```

Comment:
Hmmm, seems like this is from another commit and doesn't relate to this PR. Do you mind rebasing it?